### DISCUSSION TITLE
「开源之史」系列之十一/四：开源作为商业策略——Apache 许可的改进与‘完善’ - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#59] [Announcements]
# 「开源之史」系列之十一/四：开源作为商业策略——Apache 许可的改进与‘完善’ - 「开源之道」

https://opensourceway.community/posts/history-of-open-source/02-07-02-wired-mix-apache-and-ibm-license/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

这个故事, 以及相关故事, 其实有个隐密的线索一直没人关注,
IBM

一直就是兰色大象, 如何起舞一直是业界的话题, 却不是中心话题,
所有关注点/争论, 都在 M$ ,

可是, 各种新兴事务发韧之时, IBM 都在其中出没,
默默的...

这家造 "算盘" 起家的老公司为何如此敏锐?
是谁在其中注入了开源关注能力?

**Reply by lijiangsheng1** [reply]
是啊，除了Apache，还有对Linux的支持，也是改变历史走向的决定～  值得深挖。
